mlos_bench/mlos_bench/run.py

```diff
@@ -154,29 +123,96 @@ def _optimize(*,
     return (best_score, best_config)
 
 
-def _run(env: Environment, opt: Optimizer, trial: Storage.Trial, global_config: Dict[str, Any]) -> None:
+def _load_config(exp: Storage.Experiment, env_context: Environment,
+                 config_id: int) -> TunableGroups:
+    """
+    Load the existing tunable configuration from the storage.
+    """
+    tunable_values = exp.load_tunable_config(config_id)
+    tunables = env_context.tunable_params.assign(tunable_values)
+    _LOG.info("Load config from storage: %d", config_id)
+    if _LOG.isEnabledFor(logging.DEBUG):
+        _LOG.debug("Config %d ::\n%s",
+                   config_id, json.dumps(tunable_values, indent=2))
+    return tunables
+
+
+def _scheduler(exp: Storage.Experiment, env_context: Environment,
```

Comment:
```suggestion
def _run_schedule(exp: Storage.Experiment, env_context: Environment,
```

Though, I think this should probably be put in the "Scheduler" class I mentioned below.

Reply:
agree. Let me fix that